cdap-data-fabric/src/main/java/co/cask/cdap/data/stream/service/upload/FileContentWriter.java

```diff
@@ -54,8 +54,8 @@
     this.streamEvent = new MutableStreamEvent();
 
     directory.mkdirs();
-    this.eventFile = directory.append(String.format("upload.%s.dat", streamConfig.getStreamId()));
-    this.indexFile = directory.append(String.format("upload.%s.idx", streamConfig.getStreamId()));
+    this.eventFile = directory.append("upload.dat");
```

Comment:
The reason for keep the name in the file name is because the upload directory is shared across all streams (within the same namespace?). Having the stream name helps debugging.